code-studio/src/js/pd/workshop_dashboard/components/workshop_table.jsx

```diff
@@ -0,0 +1,109 @@
+/* global React */
+
+var WorkshopTableRow = require('./workshop_table_row.jsx');
```

Comment:
It's actually inconsistent. See https://github.com/code-dot-org/code-dot-org/tree/staging/code-studio/src/js/components (send_to_phone.jsx, share_dialog.jsx, etc). But those are in the minority. I guess this is just ruby style creep on my part. I'll update this one after the initial commit.